src/c++/library/common.h

```diff
@@ -173,7 +245,7 @@ struct InferOptions {
   /// The unique identifier for the sequence being represented by the
   /// object. Default value is 0 which means that the request does not
   /// belong to a sequence.
-  uint64_t sequence_id_;
+  SequenceId sequence_id_;
```

Comment:
I don't think this is backwards compatible. If someone has code that uses sequence_id_ as a uint64_t then I think it will fail. Can likely fix with the appropriate conversion constructors/operators.

Reply:
The SequenceId does have an operator= from uint64_t as well as from a string. I suppose they should be able to treat it just like an int or string on the client side? The only way I can think of to do this is to add set/get functions to the InferOptions struct.

Reply:
You can create a SequenceId from a uint64_t but can you assign a SequenceId to a uint64_t. I think there is C++ magic that you can add to allow that. A copy operator from SequenceId to uint64_t perhaps. You should be able to compile the original examples with you updated client code. In fact, for backwards compat testing we will want to keep some of the examples using uint64_t sequence ids

Reply:
Sure, I will just overload the Send/Stream functions to take an string parameter instead of uint64_t for each sequence example. I will look into adding the copy operator.